lib/net.js

```diff
@@ -111,7 +111,8 @@ const { isUint8Array } = require('internal/util/types');
 const {
   validateInt32,
   validatePort,
-  validateString
+  validateString,
+  validateAbortSignal
```

Comment:
Can you keep these in alphabetical order please?
```suggestion
  validateAbortSignal,
  validateInt32,
  validatePort,
  validateString